src/ResourceManager/MachineLearning/AzureRM.MachineLearning.psd1

```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.9.0' 
+ModuleVersion = '0.9.1' 
```

Comment:
@NonStatic2014 please revert this one and assemblyinfo.cs version change. We have a script that updates all the modules before release.